### PR TITLE
ref: Remove group_id column from EventAttachment

### PR DESCRIPTION
**Pull request by lynnagara**
Follow up to https://github.com/getsentry/sentry/pull/14167

### CODE REVIEW

File: src/sentry/south_migrations/0494_auto__del_field_eventattachment_group_id.py [hunk 1]
```diff
@@ -0,0 +1,1454 @@
...
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
```
Comment (wedamija) [inline src/sentry/south_migrations/0494_auto__del_field_eventattachment_group_id.py]: How high volume is this table? Should this be a dangerous migration?

Comment (lynnagara) [reply src/sentry/south_migrations/0494_auto__del_field_eventattachment_group_id.py]: I don't think it's high (this feature is currently turned on for only 16 orgs)

**Comment by lynnagara** [comment]
Note that the migration needs to be regenerated (number conflicts with https://github.com/getsentry/sentry/pull/14191)

**Comment by lynnagara** [comment]
Rebased and regenerated migration

**Review by wedamija — APPROVED** [review]
Looks ok. Probably worth watching the start of the deploy just to verify the migration goes through as expected.

File: src/sentry/south_migrations/0494_auto__del_field_eventattachment_group_id.py (hunk 1 above)
Comment (lynnagara) [reply src/sentry/south_migrations/0494_auto__del_field_eventattachment_group_id.py]: @JTCunning / @mattrobenolt Do you have any preference here around doing this manually?

Comment (lynnagara) [reply src/sentry/south_migrations/0494_auto__del_field_eventattachment_group_id.py]: Double checked this with Matt, going to leave as False

**Pull request merged**